backend/config.py

```
@@ -4,6 +4,19 @@
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    JWT_SECRET_KEY = os.environ.get("JWT_SECRET_KEY", "default_jwt_secret_key")
+    ENV = os.environ.get("FLASK_ENV", "development")
+    FRONTEND_URL = os.environ.get("FRONTEND_URL")
+
+    # SQLAlchemy connection pooling
+    SQLALCHEMY_ENGINE_OPTIONS = {
+        "pool_size": 10,
+        "max_overflow": 20,
+        "pool_timeout": 30,
+        "pool_recycle": 1800,
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)